config/routes.rb

```diff
@@ -4,7 +4,8 @@
   root 'homes#index'
   resources :users
 
-  get '/tasks', to: 'tasks#show'
+  get '/tasks' => 'tasks#show'
+  get '/groups' => 'groups#show'
```

Comment:
最近はこの書き方のほうがベターだったと思います！
```suggestion
  get '/tasks', to: 'tasks#show'
  get '/groups', to: 'groups#show'
```